src/pip/_internal/commands/wheel.py

```diff
@@ -157,7 +157,7 @@ def run(self, options, args):
         build_delete = (not (options.no_clean or options.build_dir))
         wheel_cache = WheelCache(options.cache_dir, options.format_control)
 
-        with RequirementTracker() as req_tracker, TempDirectory(
+        with get_requirement_tracker() as req_tracker, TempDirectory(
```

Comment:
I'm not sure yet if this is the right way to use this, or if we should be doing something else here or in `get_requirement_tracker`.

I was feeling too lazy to figure this out.